test/suites/integration/validation_test.go

```diff
@@ -103,13 +103,13 @@ var _ = Describe("Validation", func() {
 		})
 		It("should error when ttlSecondAfterEmpty is negative", func() {
 			nodePool.Spec.Disruption.ConsolidationPolicy = karpv1.ConsolidationPolicyWhenEmpty
-			nodePool.Spec.Disruption.ConsolidateAfter = &karpv1.NillableDuration{Duration: lo.ToPtr(-time.Second)}
+			nodePool.Spec.Disruption.ConsolidateAfter = karpv1.NillableDuration{Duration: lo.ToPtr(-time.Second)}
 			Expect(env.Client.Create(env.Context, nodePool)).ToNot(Succeed())
 		})
-		It("should error when ConsolidationPolicy=WhenUnderutilized is used with consolidateAfter", func() {
+		It("should succeed when ConsolidationPolicy=WhenUnderutilized is used with consolidateAfter", func() {
```

Comment:
Should we have a failure mode now where we try to specify either WhenUnderutilized or WhenEmpty in these integration tests and we fail when consolidateAfter isn't specified?

Reply:
We'd need to apply a real yaml to test that, right?